pf-copy-from-generated.cmd

```diff
@@ -0,0 +1,10 @@
+@echo off
+echo ========= Copying non-templated files from generated SDK back to template repository =========
+
+set SCRIPT_PATH=%~dp0
+set PF_ROOT=%SCRIPT_PATH%..\..\..
+set PF_TEMPLATE=%SCRIPT_PATH%source\
+
+set GENSDK=%PF_ROOT%\sdks\XPlatCSdk\
+
+xcopy %GENSDK% %PF_TEMPLATE% /u /s /e /y
```

Comment:
this looks to do the same thing as existing copy-from-generated.bat.  Are both needed or can you remove the old one?